hr_holidays_legal_leave/README.rst

```diff
@@ -0,0 +1,78 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+HR Holidays Legal Leave
+=======================
+
+Allows the define what holiday type is to be considered legal/annual leave type.
+As currently is Odoo assumes the type with limit=False which is a problem if
```

Comment:
Currently Odoo assumes by default leave type with limit=False. This is a problem if
you have more than one 'limited' leave type and specially confusing
when tying to set leaves from the employee form.